packages/block-library/src/embed/test/index.native.js

```diff
@@ -700,4 +730,187 @@ describe( 'Embed block', () => {
 			expect( getEditorHtml() ).toMatchSnapshot();
 		} );
 	} );
+
+	describe( 'insert via slash inserter', () => {
+		it( 'insert generic embed block', async () => {
+			const expectedHtml = EMPTY_EMBED_HTML;
+			const embedBlockSlashInserter = '/Embed';
+			const {
+				getByPlaceholderText,
+				getByA11yLabel,
+				getByText,
+			} = await initializeEditor( { initialHtml: EMPTY_PARAGRAPH_HTML } );
+
+			const paragraphText = getByPlaceholderText( 'Start writing…' );
+			fireEvent( paragraphText, 'focus' );
+			// Trigger onSelectionChange to update both the current text and text selection.
+			// This event is required by the autocompleter, as it only displays the slash inserter
+			// if the text selection is located at the end of the text, for this reason,
+			// the start and end arguments match the text length.
+			fireEvent(
+				paragraphText,
+				'onSelectionChange',
+				embedBlockSlashInserter.length,
+				embedBlockSlashInserter.length,
```

Comment:
We set the `start` and `end` offset of the text selection with the text's length to match the same behavior as when typing the text in the paragraph.